### ISSUE TITLE
Unterschied mit/ohne Index

### ISSUE THREAD

**Issue by Osslack** [Osslack/HANA_SSBM#22]
Laufzeitunterschied mit/ohne Index für Row/Column

**Comment by Pfeifenjoy** [comment]
@Osslack kannst Du das machen?

**Comment by Osslack** [comment]
Jo, Brauch dafür halt deine Analyse, dass die das pro Query kann.

Arwed Mett <notifications@github.com> schrieb am Sa., 25. März 2017, 11:44:

**Comment by Pfeifenjoy** [comment]
ah ok dann assigne ich mich mal

**Comment by Osslack** [comment]
Perfekt, wenn die Analyse das kann, geb mir Bescheid. Brauche die Daten
halt als Grundlage.

Arwed Mett <notifications@github.com> schrieb am Sa., 25. März 2017, 12:23:

**Comment by Pfeifenjoy** [comment]
@Osslack visuell ist fertig
https://github.com/Osslack/HANA_SSBM/blob/master/src/analyser/Analysis-SSBM-HANA.ipynb

**Comment by Pfeifenjoy** [comment]
So jetzt ist es auch in tabellen form

**Comment by Pfeifenjoy** [comment]
@Osslack can I close this?
